[CCUBE-2183][GZ] automated storybook component api documentation part 5 (#1356)
LifeSG/react-design-system: stories/animations/animations.mdx

Reviewer: unrelated, but this needs to be updated for V4

```suggestion
The `ThemedLoadingSpinner` automatically adapts to your theme.
```

        An animation that can be shown in between page transitions or
        while waiting for fetching of data.
      </p>

>       <p>This is a customisable animation. See the Customisable animations section below.</p>

Reviewer: ```suggestion
      <p>This is a customisable animation. See the "Customisable animations" section below.</p>
```

    </Content>
  </AnimationItem>
</AnimationDisplay>

## Customisable animations

<DocNote>
    Customisation is only supported by the animations in this section.
</DocNote>

### LoadingDotsSpinner

```tsx
import { LoadingDotsSpinner } from "@lifesg/react-design-system/animations";
```

If no color is specified, this component uses the
`Animation["loading-dots-spinner-colour"]` token.

<Canvas of={AnimationStories.CustomLoadingDotsSpinner} />

### Component tokens

> This component can be customised via the `Animation` tokens.

Reviewer: put on its own section and have a more generic description

```suggestion
## Component tokens

The `Animation` tokens allow you to customise certain aspects of specific animations.
```

<Story of={AnimationStories.TokenCustomisation} />

> ## Usage

Reviewer: can move the usage section up above "Customisable animations"

## Component API

<AutoArgTypesTabs of={AnimationStories} />

### LoadingDotsSpinner API

> <CustomisedPropsTable />

Reviewer: this can be removed
